prdoc/pr_5408.prdoc

```diff
@@ -0,0 +1,29 @@
+title: [FRAME] Cleanup storage generator types
+doc:
+  - audience: Runtime Dev
+    description: |
+      V1 pallet syntax has been deprecated and so the generator traits no longer needed, 
+      However, all methods have been ported over to `storage::types` and made public for developers
+      that require them.
+      ...
+
+crates: 
+    - name: frame-support
+      bump: minor
```

Comment:
I think those versioning are like semver, the PR can make user code not compile, I think we can do major bump I guess.

For the other crates. I guess it would also be a major bump because their dependency to frame-support is exposed into their API. But it should be automatic I guess, so no need to change them maybe.

Reply:
yeah it's not backwards compatible.